casper/src/main/scala/coop/rchain/casper/util/comm/CommUtil.scala

```diff
@@ -112,114 +110,14 @@ object CommUtil {
               }
         } yield ()
 
-      case Nil =>
-        Log[F].info(s"CASPER: Reached an end of the peers list when asking for approved block")
+      case Nil => Timer[F].sleep(delay) >> requestApprovedBlock[F](delay)
     }
 
     for {
-      a     <- MultiParentCasperConstructor[F].lastApprovedBlock
+      a     <- LastApprovedBlock[F].get
       peers <- ConnectionsCell[F].read
       local <- RPConfAsk[F].reader(_.local)
       _     <- a.fold(askPeers(peers.toList, local))(_ => ().pure[F])
     } yield ()
```

Comment:
Can we be consistent on the naming of the result of LastApprovedBlock[F].get ? Maybe lab?